gulpfile.js

```diff
@@ -127,11 +127,11 @@ gulp.task('watch', function () {
 });
 
 // Build task compile sass and twig.
-gulp.task('build', ['sass', 'twig']);
+gulp.task('build', gulp.parallel('sass', 'twig'));
 
 /**
  * Default task, running just `gulp` will compile the sass,
  * compile the project site, launch BrowserSync then watch
  * files for changes
  */
-gulp.task('default', ['browser-sync', 'watch']);
\ No newline at end of file
+gulp.task('default', gulp.series('browser-sync', gulp.series('watch')));
```

Comment:
I'm not sure if these tasks can run parallel as well; if so we could improve a bit more and change this in:

`gulp.parallel('browser-sync', 'watch')`